code/modules/research/research.dm

```diff
@@ -306,6 +307,10 @@ research holder datum.
 
 	if(current_level >= level)

```

Comment:
If you send them a disk with lower levels on it than you have previously sent, you do not get paid but the buyer's tech goes down.